apps/SageAccountWeb/src/components/ChangePasswordPage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { LeftRightPanel } from './LeftRightPanel'
+import { SourceAppLogo } from './SourceApp'
+import { SetPasswordInstructions } from './ResetPassword'
+import { ChangePassword } from 'synapse-react-client'
+
+export const ChangePasswordPage = () => {
+  const urlSearchParams = new URLSearchParams(window.location.search)
+  const errorCode = urlSearchParams.get('errorCode') ?? ''
+  const instructions =
+    errorCode == 'PASSWORD_RESET_VIA_EMAIL_REQUIRED'
+      ? 'Your current password does not meet Synapse security requirements'
+      : 'Set a new password'
+  return (
+    <>
+      <LeftRightPanel
+        className={'ResetPasswords'}
+        leftContent={
+          <>
+            <SourceAppLogo />
+            <ChangePassword />
```

Comment:
Shall we push the user back to original app (is this `appContext.redirectURL`?) once they change their password? Or alternatively push them to the root/login page and that page will redirect the user after sign-in